github/Organization.py

```diff
@@ -498,6 +503,25 @@ def create_repo(
         assert is_optional(allow_rebase_merge, bool), allow_rebase_merge
         assert is_optional(delete_branch_on_merge, bool), delete_branch_on_merge
         assert is_optional(allow_update_branch, bool), allow_update_branch
+        assert is_optional(use_squash_pr_title_as_default, bool), use_squash_pr_title_as_default
+        assert is_undefined(squash_merge_commit_title) or squash_merge_commit_title in [
+            "PR_TITLE",
+            "COMMIT_OR_PR_TITLE",
+        ], squash_merge_commit_title
```

Comment:
This can be simplified
```suggestion
        assert squash_merge_commit_title in ["PR_TITLE", "COMMIT_OR_PR_TITLE", NotSet], squash_merge_commit_title
```